### DISCUSSION TITLE
Generic mapping to existing target object

### DISCUSSION THREAD

**Discussion by aabadpci** [riok/mapperly#1357] [Q&A]
First of all I want to thank you for your awesome work and for your time.

Basically I'm trying to find a way of developing a generic method to map an object to another existing target object. I've already created a generic method that takes an object and maps it to a certain type as specified in the documentation (https://mapperly.riok.app/docs/configuration/generic-mapping/) but I can't find a way to do this with an existing target object.

The generic method I've already made is:

```csharp
public partial TTarget Map<TOrigin, TTarget>(TOrigin origin);
```

I'd like to do this also:

```csharp
// Maps origin to an existing target object
public partial void Map<TOrigin, TTarget>(TOrigin origin, TTarget target);
```
This results in the following error: "_Partial method 'method' must have an implementation part because it has accessibility modifiers._"

Is there a way of achieving this or is it planned to work in the future? 

Thanks again, kind regards!

**Comment by Smeilanbere** [comment]
Facing same issue today at work 😢

**Comment by latonz** [comment]
This is tracked in https://github.com/riok/mapperly/issues/884. Feel free to react with 👍 to the issue.

**Reply by aabadpci** [reply]
Oh, I didn't see it, my bad. Thank you for your reply!
